cli/types/tests/cypress-npm-api-test.ts

```diff
@@ -55,6 +55,33 @@ const config = defineConfig({
   modifyObstructiveCode: true
 })
 
+const solid = {
+  type: 'solid-js',
+  name: 'Solid.js',
+  package: 'solid-js',
+  installer: 'solid-js',
+  description: 'Solid is a declarative JavaScript library for creating user interfaces',
+  minVersion: '^1.0.0'
+}
+
+const thirdPartyFrameworkDefinition = defineComponentFramework({
+  type: 'cypress-ct-third-party',
+  name: 'Third Party',
+  category: 'library',
+  dependencies: (bundler) => [solid],
+  detectors: [solid],
+  supportedBundlers: ['vite', 'webpack']
+})
+
+const thirdPartyFrameworkDefinitionInvalidStrings = defineComponentFramework({
+  type: 'cypress-not-cy-wrong-prefix', // $ExpectError
+  name: 'Third Party',
+  category: 'template', // $ExpectError - only library supported for third party definitions
```

Comment:
```suggestion
  // only library supported for third party definitions
  category: 'template', // $ExpectError 
```

Looks like the syntax for dtslint is pretty strict, the extra comment following the assertion messes with this check